frontend/src/components/js/Random.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Random = () => {
```

Comment:
Consider adding PropTypes to the component to validate the props being passed to it. This can help catch bugs and improve the readability of the code. [medium]